doc/source/flexibility/extended_mysqldump.rst

```diff
@@ -50,12 +50,12 @@ Version Specific Information
 ============================
 
   * :rn:`5.7.10-1`
-    :command:`mysqldump` has been extended with :ref:`backup_locks` support
+    :command:`mysqldump` was extended with :ref:`backup_locks` support
```

Comment:
Whole "Version Specific Information" should be replaced with "8.0.12-1: feature ported from 5.7" or similar